lni.cls

```diff
@@ -305,6 +305,21 @@
 \newcommand{\authorrunning}[1]{%
    \fancyhead[LE]{\hspace{0.05cm}\oldsmall\thepage\hspace{5pt}#1}}
 \newcommand*{\email}[1]{\href{mailto:#1}{\urlstyle{same}\protect\nolinkurl{#1}}}
+\newif\iffoot\footfalse
+\let\origthanks\thanks
+\renewcommand{\thanks}[1]{\foottrue\origthanks{#1}\footfalse}
+\newcommand{\orcid}[1]{%
+	\ifstrempty{#1}{}{%
+		\href{https://orcid.org/#1}{
+			\begin{minipage}[t]{\baselineskip}
+				\includegraphics[width=\textwidth]{ORCIDiD_iconvector.pdf}
```

Comment:
I had issues with underscores (`_`) in file names.

Instead of adding a PDF, I would suggest to use [academicicons](https://ctan.org/pkg/academicons) (Source: https://tex.stackexchange.com/a/378333/9075). It is old enough to be available on overleaf and on older systems.

The alternative [orcidlink](https://ctan.org/pkg/orcidlink) is too new, even though it easier to use. (Source: https://tex.stackexchange.com/a/572014/9075)

Reply:
> Instead of adding a PDF, I would suggest to use [academicicons](https://ctan.org/pkg/academicons)

However academicons might be restricted to Xe(La)TeX or Lua(La)TeX: https://tex.stackexchange.com/questions/545599/any-way-to-use-academicons-with-pdflatex

Reply:
Then orcidlink? We can add a section to the guideline that users need to download https://github.com/duetosymmetry/orcidlink-LaTeX-command/blob/master/orcidlink.sty and place it in the current directory.

(You see I like micro libraries -- https://github.com/parro-it/awesome-micro-npm-packages ^^)

Reply:
\orcidlink{...} doesn't seem to work in the footnote: Use of \@xfootnotemark doesn't match its definition.

Also I wonder whether orcidlink can display the logo slightly higher than the preceding text, as it is done in the LNI author instructions. But I don't know how strict those instructions are.

Reply:
Thank you for checking. What do you think about basing on the TikZ macro? 😅 (I am just try to avoid an external file. Users tend to copy files back and forth - even if they should not. Copying a PDF is a thing I would like to avoid)

Reply:
I will give it a try.

Reply:
I'd prefer not to require TikZ to keep the external requirements low.

Reply:
> I prefer using the `academicons` package. However, I kindly ask for some patience.

Even though it is restricted to Xe(La)TeX or Lua(La)TeX? As far as I know, it doesn't work with pdflatex.

Reply:
For link completeness, there is also a pic2e variant: https://tex.stackexchange.com/a/659104/9075

I think, orcidlink is now the way to move forward! (Isn't it?)